modules/lambdas/main.tf

```diff
@@ -83,6 +83,7 @@ resource "aws_lambda_function" "get_current_archive_list" {
 
   environment {
     variables = {
+      SECRET_ARN = var.secret_arn
```

Comment:
Let's go with something more descriptive here and elsewhere... db_connect_into_secret_arn? Too long?

Reply:
sounds good to me